2017/SEPT/COOK86/LIKECS01/LIKECS01.cpp

```diff
@@ -0,0 +1,28 @@
+// code written by heet
+#include <iostream>
+#include <cstring>
+
+using namespace std;
+
+int main() {
+	int t;
+	cin >> t;
+	while (t--) {
+		string s;
+		cin >> s;
+		int a[26] = {0};
+		int n = s.size();
+		bool isbreak = false;
+		for (int i = 0; i < n; ++i) {
+			a[(int) s[i] - 97]++;
```

Comment:
The space after `(int)` ? No need for that please remove it.

Reply:
isn't ```s[i]``` returns character at i'th position of string, if so then it can show error.

Reply:
@Hsankesara I'm talking about the "space" after `(int)` , when did I point anything about `s[i]`?

Reply:
@Hsankesara remove the space.

Reply:
sorry read it wrong...I'll change it